docker/.env.example

```
@@ -606,7 +606,7 @@ INDEXING_MAX_SEGMENTATION_TOKENS_LENGTH=1000
 INVITE_EXPIRY_HOURS=72
 
 # Reset password token valid time (hours),
-RESET_PASSWORD_TOKEN_EXPIRY_HOURS=0.0833
+RESET_PASSWORD_TOKEN_EXPIRY_HOURS=1
```

```suggestion
RESET_PASSWORD_TOKEN_EXPIRY_HOURS=24
```

Leave it as default value `24`.